develop/toolkit/api-libraries/polkadot-js-sdk.md

```diff
@@ -1,4 +1,109 @@
 ---
 title: Polkadot.js SDK
-description: TODO
----
\ No newline at end of file
+description: Interact with Polkadot SDK-based chains easily using the Polkadot.js SDK. Query chain data, submit transactions, and more via JavaScript.
+---
+
+# Polkadot.js SDK
+
+## Introduction
+
+The Polkadot.js SDK uses JavaScript/Typescript to interact with Polkadot SDK-based chains. It allows you to query nodes, read chain state, and submit transactions through a dynamic, auto-generated API interface.
+
+### Dynamic API Generation
+
+Unlike traditional static APIs, the Polkadot.js API generates its interfaces automatically when connecting to a node. Here's what happens when you connect:
+
+1. The API connects to your node
+2. It retrieves the chain's metadata
+3. Based on this metadata, it creates specific endpoints in the format - `api.<type>.<module>.<section>`
+
+### Available API Categories
+
+You can access three main categories of chain interactions:
+
+1. **Constants** (`api.consts`)
+    - Access runtime constants directly
+    - Returns values immediately without function calls
+    - Example - `api.consts.balances.existentialDeposit`
+
+2.  **Queries** (`api.query`)
+    - Read chain state
+    - Example - `api.query.system.account(accountId)`
+
+3. **Transactions** (`api.tx`)
+    - Submit extrinsics (transactions)
+    - Example - `api.tx.balances.transfer(accountId, value)`
+
+The available methods and interfaces will automatically reflect what's possible on your connected chain.
+
+## Installation
+
+To add the Polkadot.js SDK to your project:
```

Comment:
There is not a single mention of https://github.com/polkadot-js/api, its dedicated docs: https://polkadot.js.org/docs/api/, and the installation section therein: https://polkadot.js.org/docs/api/start/install

You should contribute back to these sources, and only add what is a high level overview here.

Reply:
@0xLucca Agree let's ensure we link to the official docs.

@kianenigma it is unclear wether these docs are maintained as the last commit to it was 7 months ago